ambry-clustermap/src/test/java/com.github.ambry.clustermap/ClusterManagerWriteStatusDelegateTest.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
this file can be removed now

Reply:
It's removed locally and claims to be in-sync with the origin so I'm not sure why it still shows up here....